content/en/gitsign/overview.md

```diff
@@ -8,16 +8,13 @@ position: 150
 <img src="/sigstore_overview_v1.jpg" class="light-img" width="1280" height="640" alt=""/>
 
 Gitsign implements keyless Sigstore to sign Git commits with a valid [OpenID](https://openid.net/connect/) identity.
-In practice, that means you won't need to deal with gpg keys and a complicated setup in order to sign your Git commits. After installing and configuring Gitsign within your project, when signing your commits you will be redirected to a browser window to authenticate with a supported OpenID provider, such as GitHub or Google.
-Signing details will then be stored in [Rekor](/rekor/overview) for posterior consultation.
+In practice, that means you won’t need GPG keys and a complicated setup in order to sign your Git commits. After installing and configuring Gitsign within your project and signing your commits, you will be redirected to a browser window to authenticate with a supported OpenID provider, such as GitHub or Google. Signing details will then be stored in [Rekor](/rekor/overview) for subsequent consultation.
```

Comment:
As far as I understood, redirecting to a browser window happens right when you sign (after running the command), for every commit. I think the rephrase here make it sounds like the redirection happens after signing